packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -51,8 +58,13 @@ export type Options = {
 );
 
 export class MetriportMedicalApi {
+  // TODO this should be private
```

Comment:
Because its a breaking change. I don't like adding TODOs without real action, but considering:
- it's a breaking change
- doesn't really impact the platform
- we'll replace this w/ a Fern generated one

I thought just adding the TODO for context when we do a larger overhaul of the SDK.

Reply:
Ah ok is there a ticket for it then with context? Just didn't really know what it's about when looking at it

Reply:
I don't think there's an action here, this is mostly to flag a design issue that we can address when we overhaul the SDK.